src/licensemanager2/workload_managers/slurm/cmd_utils.py

```diff
@@ -226,6 +238,10 @@ async def scontrol_show_lic():
     )
 
     stdout, _ = await asyncio.wait_for(proc.communicate(), CMD_TIMEOUT)
+    if proc.returncode != 0:
+        msg = "Scrontrol process return value was different from 0"
```

Comment:
```suggestion
        msg = f"Scontrol process returned {proc.returncode}"
```

What about also showing in the logs the return code *and* the stdout/stderr? Knowing why the command failed is very important for debugging.